python/pyspark/sql/tests/connect/test_parity_python_streaming_datasource.py

```diff
@@ -0,0 +1,35 @@
+#
```

Comment:
should list this file into https://github.com/apache/spark/blob/master/dev/sparktestsupport/modules.py

Reply:
Added.